appcues/src/main/java/com/appcues/AppcuesConfig.kt

```diff
@@ -90,6 +90,12 @@ public data class AppcuesConfig internal constructor(
      */
     var additionalAutoProperties: Map<String, Any> = emptyMap()
 
+    /**
+     * Optional property that allows for setting custom package list for fonts, mostly used when your fonts are stored in
+     * a different package than your main app (multi-module apps)
+     */
+    var packageNames: List<String> = arrayListOf()
```

Comment:
should it be something more specific like `fontPackageNames` ?

Reply:
so I was considering leaving this as packageNames for in case in the future we need this for something else besides fonts.